index.html

```diff
@@ -47,119 +53,131 @@ <h4>LET'S CONNECT</h4>
+                        <li class="element2">Back End Dev</li>
+                        <li><img src="Counter.svg" alt="dot"></li>
+                        <li class="element2">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                    <ul class="bottom-list">
+                        <li><button class="language-buttons" type="button">html</button></li>
+                        <li><button class="language-buttons" type="button">css</button></li>
+                        <li><button class="language-buttons" type="button">javaScript</button></li>
+                    </ul>
+                    <button class="p-button" type="button">See project</button>
+                </div>
             </div>
             <div class="flex-card">
                 <img class="snapshot" src="Snapshoot4Portfolio.svg" alt="portfolio 1">
-                <h1>Multi-Post Stories</h1>
-                <ul>
-                    <li class="element1">CANOPY</li>
-                    <li><img src="Counter.svg" alt="dot"></li>
-                    <li class="element2">Back End Dev</li>
-                    <li><img src="Counter.svg" alt="dot"></li>
-                    <li class="element2">2015</li>
-                </ul>
-                <p>A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required.</p>
-                <ul class="bottom-list">
-                    <li><button class="language-buttons" type="button">html</button></li>
-                    <li><button class="language-buttons" type="button">css</button></li>
-                    <li><button class="language-buttons" type="button">javaScript</button></li>
-                </ul>
-                <button class="p-button" type="button">See project</button>
+                <div class="column2">
+                    <h1>Multi-Post Stories</h1>
+                    <ul>
+                        <li class="element1">CANOPY</li>
+                        <li><img src="Counter.svg" alt="dot"></li>
+                        <li class="element2">Back End Dev</li>
+                        <li><img src="Counter.svg" alt="dot"></li>
+                        <li class="element2">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                    <ul class="bottom-list">
+                        <li><button class="language-buttons" type="button">html</button></li>
+                        <li><button class="language-buttons" type="button">css</button></li>
+                        <li><button class="language-buttons" type="button">javaScript</button></li>
+                    </ul>
+                    <button class="p-button" type="button">See project</button>
```

Comment:
Kindly update the button hover to show the text in white as shown in Figma design 😃

Your design:

![image](https://user-images.githubusercontent.com/73714615/188193398-69767da5-6f91-4246-b6de-93f48cac8abf.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188193537-43e7692f-b6cd-4793-8047-66d389e53c2d.png)